crates/turbopack-build/src/chunking_context.rs

```diff
@@ -243,6 +255,25 @@ impl ChunkingContext for BuildChunkingContext {
         self.environment
     }
 
+    #[turbo_tasks::function]
+    async fn asset_url(self: Vc<Self>, ident: Vc<AssetIdent>) -> Result<Vc<String>> {
+        let this = self.await?;
+        let asset_path = ident.path().await?.to_string();
+        let asset_path = asset_path
+            .strip_prefix(&format!("{}/", this.client_root.await?.path))
+            .context("expected client root to contain asset path")?;
+
+        Ok(Vc::cell(format!(
+            "{}{}",
+            this.asset_prefix
+                .await?
+                .as_ref()
+                .map(|s| s.to_owned())
+                .unwrap_or_else(|| "/".to_owned()),
```

Comment:
This doesn't need to become an owned value (and clones the string).

That should do it:

``` rs
.as_ref()
..unwrap_or("/")
```